.gitignore

```diff
@@ -28,6 +28,7 @@
 *.safetensors
 *.gguf
 *.vmfb
+iree_tests/onnx/node/generated/test_*/iree_output_*.npy
```

Comment:
The `iree_tests/` folder has its own `.gitignore` file to keep the top level file cleaner: https://github.com/nod-ai/SHARK-TestSuite/blob/main/iree_tests/.gitignore